pkg/lendertesting/lend_test.go

```diff
@@ -262,7 +283,67 @@ func (s *lendTestSuite) TestLendSwivel() {
 	signatureResult, err = s.Swivel.InitiateCalledSignature(ORDERS[1].Maker)
 	assert.Nil(err)
 	assert.Equal(COMPONENTS[1].V, signatureResult)
+}
+
+func (s *lendTestSuite) TestLendElement() {
+	assert := assert.New(s.T())
+
+	s.MarketPlace.MarketsReturns([8]common.Address{
```

Comment:
we can always make a helper that just returns 8 addresses vs doing this in every test